tests/test_resource_realtimeapi.py

```diff
@@ -25,7 +25,7 @@ def _get_class_path(clsname):
         return '{}.{}.{}'.format(__package__, module_name, clsname)
 
     def test_realtimeapi_with_default_settings(self):
-        expected_entities = {'crawl.json': CrawlResource}
+        expected_entities = {b'crawl.json': CrawlResource}
```

Comment:
tests in `test_resource_realtime` are checking whether resource names are matched to resource object. Resource names must be bytes for Twisted, so we must keep b'' in tests. If resource name is string, not bytes it will return 404 in python 3. does it make sense?